nodejs/week1/meals/src/backend/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 3000;
+
+// Routers - Inside the index.js file create routes using express
+app.get("/", (req, res) => res.send("Hello World"));
+
+const meals = require("./routes/meals.js");
+app.get("/meals", meals);
+
+const meal = require("./routes/meal.js");
+app.get("/meal", meal);
+
+const cheapMeals = require("./routes/cheap-meals.js");
+app.get("/cheap-meals", cheapMeals);
+
+const largeMeals = require("./routes/large-meals.js");
+app.get("/large-meals", largeMeals);
+
+const reservation = require("./routes/reservation.js");
+app.get("/reservation", reservation);
+
+const reservations = require("./routes/reservations.js");
+app.get("/reservations", reservations);
+
+app.listen(port, () => console.log(`meal app is listening on port ${port}`));
```

Comment:
By convention we put all the imports (requires) at the top of the file. Otherwise it is hard to understand what is going on in a file because imports are intermingled with code. So use:

// ------- imports/requires ---------------
// these are all the things I need in this file that
// come from other files

// -------- actual code --------
// this is the purpose of this file. Why this file exists
// it is the most relevant part of the file. That is why
// it is specially important that it is super tidy and clear
A nice metaphor are ingredients and a recipe. The ingredients are what you need to make something work, and the recipe are the steps you follow to create a dish using the ingredients above. The imports/requires are your ingredients. The code that you write is your recipe. Ingredients first. Then use the ingredients in your recipe :)